app/services/user-role-service.js

```diff
@@ -56,9 +63,11 @@ module.exports.removeUserByUsername = function (username) {
 }
 
 var removeDomainFromUsername = function (username) {
```

Comment:
We could expose this as a public method, unit test it and use it in `app/authentication`.

Reply:
done